src/util.jl

```diff
@@ -66,3 +66,95 @@ function zero_crossings(array::AbstractVector{T}, size::Int = length(array), off
     end
     result
 end
+
+# units from SampledSignals
+
+const frames = Hz*s
+const FrameQuant = DimensionlessQuantity
```

Comment:
SInce Unitful.jl is no longer a dependency, where do these come from?

Reply:
Unit full is still there 😅